spring-cloud-azure-samples/eventhubs-binder-sample/README.adoc

```diff
@@ -112,6 +112,50 @@ spring.cloud.azure.eventhub.checkpoint-storage-account=[checkpoint-storage-accou
 spring.cloud.azure.eventhub.namespace=[eventhub-namespace]
 ....
 
+=== Steps for application to run in App Service
```

Comment:
Could you remove the duplicate part with vm?

Reply:
Sure, then the two parts will mix together